tasks/rh-sign-image-cosign/tests/test-rh-sign-image-cosign-single-component.yaml

```diff
@@ -0,0 +1,78 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-rh-sign-image-cosign-single-component
+spec:
+  description: Test signing a single image by cosign
+  workspaces:
+    - name: tests-workspace
+  tasks:
+    - name: setup
+      workspaces:
+        - name: data
+          workspace: tests-workspace
+      taskSpec:
+        workspaces:
+          - name: data
+        steps:
+          - name: setup-values
+            image: quay.io/konflux-ci/release-service-utils:38c3bfd7479c86b832cba5b61f9bbde40c469393
+            script: |
+              #!/usr/bin/env bash
+              set -eux
+              _TEST_MANIFEST_REFERENCE="quay.io/redhat-pending/test-product----test-image0@sha256:2222"
+              _TEST_MANIFEST_REPO="quay.io/redhat-pending/test-product----test-image0"
+              cat > $(workspaces.data.path)/snapshot_spec.json << EOF
+              {
+                "application": "myapp",
+                "components": [
+                  {
+                    "name": "comp0",
+                    "containerImage": "$_TEST_MANIFEST_REFERENCE",
+                    "repository": "$_TEST_MANIFEST_REPO"
+                  }
+                ]
+              }
+              EOF
+    - name: run-task
+      taskRef:
+        name: rh-sign-image-cosign
+      params:
+        - name: snapshotPath
+          value: snapshot_spec.json
+        - name: secretName
+          value: 'test-cosign-secret'
+      workspaces:
+        - name: data
+          workspace: tests-workspace
+      runAfter:
+        - setup
+    - name: check-result
+      workspaces:
+        - name: data
+          workspace: tests-workspace
+      taskSpec:
+        steps:
+          - name: check-result
+            image: quay.io/konflux-ci/release-service-utils:38c3bfd7479c86b832cba5b61f9bbde40c469393
+            script: |
+              #!/usr/bin/env bash
+              set -eux
+              echo "check results"
+              _TEST_PUB_REPO1="registry.stage.redhat.io/test-product/test-image0"
+              _TEST_PUB_REPO2="registry.access.stage.redhat.com/test-product/test-image0"
+              _TEST_REPO="quay.io/redhat-pending/test-product----test-image0"
+
+              CALLS=$(cat "$(workspaces.data.path)/mock_skopeo_calls")
+              test "$CALLS" = "inspect --raw docker://${_TEST_REPO}@sha256:2222"
+              CALLS=$(cat "$(workspaces.data.path)/mock_cosign_calls")
```

Comment:
Can you add an empty line above? To visually separate the skopeo calls check from the cosign calls check.

Reply:
Fixed.